tool/tctl/common/db_command.go

```diff
@@ -97,7 +97,7 @@ Fill out and run this command on a node to start proxying the database:
    --auth-server={{.auth_server}} \
    --name={{.db_name}} \
    --protocol={{.db_protocol}} \
-   --uri={{.db_uri}}
+   --uri={{.db_uri}}:5432
```

Comment:
Other examples use the port: https://github.com/gravitational/teleport/blob/9410346b8d0dae04288847be5c7bf90b95fed4bd/tool/teleport/common/usage.go#L34-L47

If I copy and paste the current output I get an error:

```
...
--uri=35.188.22.222
ERROR: invalid database "postgres" address "35.188.22.222": address 35.188.22.222: missing port in address
```

Reply:
Reverted and verified that all examples include `port`. The `port` must be included and it's consistent throughout our docs.